compile.go

```diff
@@ -40,16 +40,12 @@ func Parse(flux string) (*ast.Package, error) {
 // Eval accepts a Flux script and evaluates it to produce a set of side effects (as a slice of values) and a scope.
 func Eval(ctx context.Context, flux string, opts ...ScopeMutator) ([]interpreter.SideEffect, values.Scope, error) {
 	h := parser.ParseToHandle([]byte(flux))
-	return evalHandle(ctx, h, opts...)
+	return defaultRuntime.evalHandle(ctx, h)
```

Comment:
`opts` appears to be unused?

Reply:
I need to figure out what opts is used for and use it. I am unclear on exactly how it is used so I"ll circle back on this.

Reply:
Fixed. It turns out it was easier than I thought it would be.